### ISSUE TITLE
Alpha Range or Character Grouping Request

### ISSUE THREAD

**Issue by mmheller** [adiwg/mdEditor#177]
In the near future when the Metadata Records, Contacts, and Dictionaries list sections on the left mdEditor panel get populated with many records, it would be nice to enhance the UI to handle many records for easier browsing/selecting.  We have talked about the possibility of adding functionality to group records based on Alpha Ranges (i.e. A-F, G-L, etc.) or single Alpha character.  The range groupings would only take effect if the number of records would be greater than a given number (e.g. 30).  This range grouping functionality concept is similarly seen in music browsing apps.  When clicking on a range or character the records that match would display for selection.

![image](https://user-images.githubusercontent.com/6976696/38056249-3e41147a-3299-11e8-907f-613c6353ba18.png)

**Comment by tpatterson1996** [comment]
yes, please!  I have a long list of projects, products, and contacts.  It would be greatly appreciated if there were alphabetized and possibly filtered by administrator or other contact field :)
